### DISCUSSION TITLE
Informe pregunta 1. parte 1)

### DISCUSSION THREAD

**Discussion by TarekHirmas** [IIC2133-PUC/2024-2#192] [Tarea 3] [answered]
Holaa, en la pregunta "Describe cómo sería una solución utilizando un enfoque "Greedy" ". Se debe mostrar en el informe que la solución greedy llega a una óptima global?. (Si es así, es necesario mostrarlo con inducción? o puede ser con palabras justificandolo?). Graciaaas.

**Comment by tomastrivino** [comment, marked as answer]
Dado el límite de páginas, basta que lo justifiques adecuadamente usando palabras. Si decides usar inducción, asegúrate que no sea algo demasiado extenso, ya que tienes que argumentar otras cosas en tu informe.
